src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.php

```diff
@@ -218,7 +218,7 @@
                 inline_service()
                     ->factory([SerializerErrorRenderer::class, 'getPreferredFormat'])
                     ->args([service('request_stack')]),
-                service('error_renderer.html'),
+                service('error_renderer.default'),
```

Comment:
this should make Symfony generate a lazy proxy, so that we don't create the service unless it's used:
```suggestion
                inline_service(ErrorRendererInterface::class)
                    ->factory('current')
                    ->args([['error_renderer.default']])
                    ->lazy(),
```

Reply:
Updated with proper syntax.